collections/seasonal-trajectories.yaml

```diff
@@ -0,0 +1,120 @@
+Name: Seasonal Trajectories, 10-daily
+Description: |
+ Seasonal Trajectories (ST) product is a filtered time series of [Plant Phenology Index(PPI)](https://www.nateko.lu.se/research/remote-sensing-and-earth-observation/lund-earth-observation-research-group/vegetation-phenology) provided yearly on a 10-daily basis at 10m resolution . 
+ It is part of the Copernicus Land Monitoring Service (CLMS), pan-European High Resolution Vegetation Phenology and Productivity [(HR-VPP)](https://land.copernicus.eu/pan-european/biophysical-parameters/high-resolution-vegetation-phenology-and-productivity) product suite. 
+ The Seasonal Trajectories PPI is derived through fitting a smoothing and gap filling function to the yearly time-series raw PPI values generated from Sentinel-2 satellite observations. 
+ In addition to the PPI band, a  QFLAG band  indicating the quality of the smoothing process is included. 
+ The Seasonal Trajectories  provide the vegetation status for each pixel on a regular 10-day time step from January 1 2017 onwards over the  EEA39 region. 
+ More information about ST product is outlined in the [product user manual](https://land.copernicus.eu/user-corner/technical-library/product-user-manual-of-seasonal-trajectories/)  
+AdditionalInfoExternal:
+   Title: Additional info 
+   Path: seasonal-trajectories/README.MD 
+Image: seasonal-trajectories/thumbnail.png 
+Resolution: 10m
+GeographicalCoverage: Europe (Lat; 26N, 72N Lon; -25 W 45 E)
+TemporalAvailability: January 1, 2017 - ongoing
+UpdateFrequency: Annually
+BandInformation: 
+  Table:
+      Columns:
+        - Name: Name
+          Title: Name
+        - Name: Description
+          Title: Description  
+        - Name:  Unit
```

Comment:
I use VS code  and apparently it does not alert double spaces,   
which editor would you reccomend ?

Reply:
I also use VS code and it's just fine. I didn't check for any extension that would alert double spaces but if you do a search for double space it will highlight them in the file and it's very easy to remove them (I guess that's what you did).